exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -589,8 +589,147 @@ func (p *Poloniex) CancelAllOrders(_ *order.Cancel) (order.CancelAllResponse, er
 
 // GetOrderInfo returns order information based on order ID
 func (p *Poloniex) GetOrderInfo(orderID string, pair currency.Pair, assetType asset.Item) (order.Detail, error) {
-	var orderDetail order.Detail
-	return orderDetail, common.ErrNotYetImplemented
+	var orderInfo order.Detail
+	trades, err := p.GetAuthenticatedOrderTrades(orderID)
+	if err == nil {
+		orderInfo.Exchange = p.Name
+		orderInfo.Pair = pair
```

Comment:
Can we move the assignments outside of this scope?

Reply:
GetAuthenticatedOrderTrades returns "Order not found" on open orders, so we will lose info from GetAuthenticatedOrderStatus  and vice versa - GetAuthenticatedOrderStatus returns  "Order not found"  on closed orders

Reply:
"Can we move the assignments outside of this scope?" - done